### PR TITLE
Change login flow to use Ory

### PR DESCRIPTION
**Pull request by mpgxvii**
Description: 
- Converts the management portal web app login to use the auth code login flow
- Adds support for `Hydra` oauth tokens by adding it as a `JwksTokenVerifier`
- Adds auth server (hydra) config
- Move all security config to `SecurityConfiguration` class and remove `OAuth2ServerConfiguration` class
- Removes unused security, config, and oauth classes

Fixes:
- https://github.com/RADAR-base/hydra-kratos/issues/19

#### Checklist:
- [ ] The [Main workflow](https://github.com/RADAR-base/ManagementPortal/actions/workflows/main.yml) has succeeded
- [ ] The [Gatling tests](https://github.com/RADAR-base/ManagementPortal#other-tests) have passed
- [ ] I have logged into the portal running locally with default admin credentials
- [ ] I have updated the README files if this change requires documentation update
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/main/java/org/radarbase/management/web/rest/LoginEndpoint.kt [hunk 1]
```diff
@@ -1,36 +1,112 @@
...
+            return "${config.authServer.serverUrl}/oauth2/auth?" +
+                    "client_id=${config.frontend.clientId}&" +
+                    "response_type=code&" +
+                    "state=${Instant.now()}&" +
+                    "audience=res_ManagementPortal&" +
+                    "scope=offline&" +
+                    "redirect_uri=$mpUrl/api/redirect/login"
+        }
+
+        private suspend fun fetchAccessToken(
```
Comment (mpgxvii) [inline src/main/java/org/radarbase/management/web/rest/LoginEndpoint.kt:80]: This should be moved out to a service.

**Review by Bdegraaf1234 — APPROVED** [review]
I think this means we can delete KratosTokenVerifierLoader

Probably good to do asap, or at least put it on a tasklist somewhere.

Otherwise, LGTM :)

**Comment by Bdegraaf1234** [comment]
The output of  `gradlew check`  does not show, but the test does not pass. Do you know why?

**Comment by mpgxvii** [comment]
> The output of `gradlew check` does not show, but the test does not pass. Do you know why?

I've updated this now 👍

**Review by yatharthranjan — APPROVED** [review]
LGTM

**Pull request merged**